Hardcode/vision_image.py

```diff
@@ -0,0 +1,68 @@
+''' 
+******************** Pipeline driving code ******************
+Run:
+
+python vision_image.py \
+	-i ../../datasets/RoboMasterLabelledImagesSet1/image-550.jpg \
+	--save ./output/
+
+'''
```

Comment:
Having a shell script with this command in this folder may be helpful

Reply:
Yes I agree, I will make one